app/utils/handleQualification/index.ts

```diff
@@ -0,0 +1,2 @@
+export const isPlayoff = (i: number) => i <= 5
+export const isPlayin = (i: number) => i >= 6 && i < 10
```

Comment:
Just to keep the convention from other functions:

```suggestion
export function isPlayoff(i: number) {
  return i <= 5
}

export function isPlayin(i: number) {
  return i >= 6 && i < 10
}
```

Reply:
I believe he took from my suggestion, so it's my fault if I wrote it different than our conventions hahaha